src/com/hrabrov/introduction_to_java/algorithmization/README.md

```diff
@@ -0,0 +1,8 @@
+# Task for algorithmization
+
+There is contained tasks on Algorithmization:
+- one-dimensional arrays;
+- multidimensional arrays;
+- sorting one-dimensional array;
```

Comment:
* `Sorted` one-dimensional ... 
or 
* `Sorting of` ...
Depends of the intended meaning